src/esp/assets/ResourceManager.cpp

```diff
@@ -1783,10 +1783,21 @@ bool ResourceManager::loadRenderAssetGeneral(const AssetInfo& info) {
   // Set transformations. Objects that are not part of the hierarchy are
   // ignored, nodes that have no transformation entry retain an identity
   // transformation.
-  for (const Cr::Containers::Pair<unsigned, Mn::Matrix4>& transformation :
+  for (Cr::Containers::Pair<unsigned, Mn::Matrix4>& transformation :
        scene->transformations3DAsArray()) {
     if (Cr::Containers::Optional<esp::assets::MeshTransformNode>& node =
             nodes[transformation.first()]) {
+      if (transformation.second().scaling().product() == 0) {
+        ESP_ERROR() << "0 value in scaling vector for mesh transform. "
+                       "Indicates an issue with source asset. Overriding to "
+                       "small nonzero and continuing.";
+        ESP_ERROR() << transformation.second();
+        for (auto ix = 0; ix < 3; ++ix) {
+          if (transformation.second()[ix][ix] == 0) {
```

Comment:
```suggestion
          if (transformation.second()[ix].xyz().isZero()) {
```

could properly handle also cases with rotations? i.e., a matrix that's rotating by 90° on Y has zeros on diagonal, but is still valid:

```py
>>> from magnum import *
>>> Matrix4.rotation_y(Deg(90))
Matrix(0, 0, 1, 0,
       0, 1, 0, 0,
       -1, 0, 0, 0,
       0, 0, 0, 1)

```